app/models/ability.rb

```diff
@@ -5,7 +5,8 @@ def initialize(user)
     group_member_data = [Example, LingsProperty, ExamplesLingsProperty]
     group_admin_data  = [Ling, Property, Category, Membership]
 
-    group_expert_data = [Ling, Example, LingsProperty, ExamplesLingsProperty]
+    group_expert_data = [LingsProperty, ExamplesLingsProperty, Ling]
```

Comment:
doesn't need to have Example in this group.
If the user can edit a ling, he can also add, edit and remove examples